conf/config.neon

```diff
@@ -121,6 +121,7 @@ parameters:
 	reportMaybesInPropertyPhpDocTypes: false
 	reportStaticMethodSignatures: false
 	reportWrongPhpDocTypeInVarTag: false
+	stricterPhpDocTypeCheckInVarTag: false
```

Comment:
`stricter` isn't really descriptive. It should say something that it always requires a subtype...

Reply:
what about 

- `reportAnyWideningInPhpDocTypeInVarTag` (the new one)
- `reportWrongPhpDocTypeInVarTag`

Reply:
What about `reportAnyTypeWideningInVarTag`? `PhpDocType` is redundant and not always true, because the behaviour depends on `reportWrongPhpDocTypeInVarTag` being true or false.

Reply:
Ok then, `reportAnyTypeWideningInVarTag`

Reply:
renamed